src/api/routes/healthy.ts

```diff
@@ -1,17 +1,17 @@
 import type express from 'express';
 
-import isReady from 'lib/helpers/isReady';
+import { isReady } from 'lib/helpers/isReady';
 import renderer from 'lib/rendererSingleton';
 
 let readyOnce = false;
 
-export default async function healthy(
+export async function healthy(
   req: express.Request,
   res: express.Response
 ): Promise<void> {
   // The system cannot be checked for healthyness unless it has already started
   if (!readyOnce) {
-    readyOnce = await isReady();
+    readyOnce = isReady();
   }
   if (!readyOnce) {
     res.status(200).send();
```

Comment:
There is no return after send, is this wanted ?

Reply:
good catch indeed